nncf/torch/graph/node_utils.py

```diff
@@ -0,0 +1,44 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+from torch import Tensor
+from nncf.common.graph.graph import NNCFNode
+from nncf.torch.nncf_network import NNCFNetwork
+from nncf.torch.graph.operator_metatypes import OPERATORS_WITH_BIAS_METATYPES
+
+
+def is_node_with_bias(node: NNCFNode, model: NNCFNetwork) -> bool:
+    """
+    Checks if the node has a bias or not.
+
+    :param node: The node to check.
+    :param model: The model that contains this operation.
+    :return: Return `True` if `node` corresponds to the operation
+        with bias (bias is added to the output tensor of that operation),
+        `False` otherwise.
+    """
+    node_module = model.get_containing_module(node.node_name)
+    return node.metatype in OPERATORS_WITH_BIAS_METATYPES and node_module.bias is not None
```

Comment:
As I know, some values are not biased because the wrong shape for the "bias" constant exists.
@kshpv, have found it previously. Should we check it here too?

Reply:
After a discussion with @alexsu52 we decided that we should update all biases, neither their shape